interactions/api/gateway/client.pyi

```diff
@@ -6,9 +6,8 @@ from asyncio import (
 from logging import Logger
 from typing import Any, Dict, List, Optional, Tuple, Union, Iterable
 
-from aiohttp import ClientWebSocketResponse
+from aiohttp import ClientWebSocketResponse, WSMessage
 
-from .heartbeat import _Heartbeat
```

Comment:
I'm against the removal of the heartbeat class, please revert this changes

Reply:
??
What's the explanation behind this comment?

Reply:
I don't see the reason for a removal, imo it is an unnecessary change